docs/api-docs/getting-started/best-practices.md

```diff
@@ -20,7 +27,7 @@ BigCommerce frequently enhances its core product and is actively developing v3 A
 
 ## Use Webhooks Effectively
 
-To keep data in your application up-to-date, [webhooks](/api-docs/getting-started/webhooks/about-webhooks) provide a great alternative to doing periodic checks. In order to register a webhook event that your application can listen for, you will need to use OAuth (not legacy “Basic Authentication”).
+To keep data in your application up-to-date,[webhooks](/api-docs/getting-started/webhooks/about-webhooks) provide a great alternative to doing periodic checks. In order to register a webhook event that your application can listen for, you will need to use OAuth (not legacy “Basic Authentication”).

```

Comment:
**Suggestion:** Use OAuth to register a webhook event that your application can listen for. Do not use legacy “Basic Authentication.“